src/status_im/chat/styles/input/emoji.cljs

```diff
@@ -0,0 +1,16 @@
+(ns status-im.chat.styles.input.emoji
+  (:require [status-im.components.styles :as common]))
```

Comment:
it depends — sometimes we also use `:as`. 
I think that it's bad to use `:refer` and we should use `:as` instead *everywhere*, so I decided to use it here as an example.